apps/ftu/js/navigation.js

```diff
@@ -246,6 +249,16 @@ var Navigation = {
             sharePerformance.checked = req.result[settingName] || false;
           };
         }
+
+        // If it's a dogfooder, we don't want them to disable the metrics.
+        var dogfood = settings && settings.createLock().get(DOGFOODSETTING);
+        if (dogfood) {
+          dogfood.onsuccess = function() {
+            if(dogfood.result[DOGFOODSETTING] === '1') {
```

Comment:
seems weird to store a `1` as the flag value. Is there a spec for it os is it arbitrary?